CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## [7.0.0](https://github.com/theforeman/puppet-qpid/tree/7.0.0) (2021-01-28)
+
+[Full Changelog](https://github.com/theforeman/puppet-qpid/compare/6.1.0...7.0.0)
+
+**Implemented enhancements:**
+
+- Fixes [\#30252](https://projects.theforeman.org/issues/30252): Make wait for port more resilient [\#137](https://github.com/theforeman/puppet-qpid/pull/137) ([ehelms](https://github.com/ehelms))
```

Comment:
This was already released as part of 6.2.0. Please include the changelog parts from a1a361fef88941f788e3db585debda1b43ef9d4d.

Reply:
I'd drop this section since it's the same as 6.2.0. So effectively we do a major version just for the modern facts. That's why I've been holding off on a release last time: for the end user it doesn't really make a difference.